cfn.template.yaml

```diff
@@ -218,3 +224,89 @@ Resources:
             - '*'
       Roles:
         - !Ref deliveryRole
+
+  # ALB Solution
+  LambdaPermissionForALB:
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt Lambda.Arn
+      Action: lambda:InvokeFunction
+      Principal: elasticloadbalancing.amazonaws.com
+      SourceArn: !Ref TargetGroupForLambda
+
+  ConsentLoadBalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties: 
+      IpAddressType: ipv4
+      LoadBalancerAttributes:
+        - Key: idle_timeout.timeout_seconds
+          Value: 10
+      Name: !Sub 
+        - ${Stack}-${App}-${Stage}
+        - App:
+            !FindInMap [Constants, App, Value]
+          Stack:
+            !FindInMap [Constants, Stack, Value]
+      Scheme: internet-facing
+      SecurityGroups: 
+        - !Ref AllowHttpSecurityGroup
+      Subnets: !Ref FrontendSubnets
+      Tags: 
+        - { Key: Stack, Value: !FindInMap [Constants, Stack, Value] }
+        - { Key: App, Value: !FindInMap [Constants, App, Value] }
+        - { Key: Stage, Value: !Ref Stage }
+      Type: application
+
+  TargetGroupForLambda:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      Matcher: 
+        HttpCode: 200
```

Comment:
Would 200-299 be better?

(Or 200-399 if we want to allow all the HTTP status codes that represent good results)

In this case we might know our Lambda will only return 200s, but in that case I'd be tempted to leave a comment so that people that copy this template understand 200 isn't *generally* the correct choice for a web-lambda.